projects/appimagekit.cmake

```diff
@@ -20,4 +22,6 @@ if(BUILD_OS_LINUX)
         COMMAND ${CMAKE_COMMAND} -E copy ${CMAKE_BINARY_DIR}/AppImageKit-prefix/src/AppImageKit/AppImageMonitor   ${CMAKE_INSTALL_PREFIX}/bin/
         COMMAND ${CMAKE_COMMAND} -E copy ${CMAKE_BINARY_DIR}/AppImageKit-prefix/src/AppImageKit/AppRun            ${CMAKE_INSTALL_PREFIX}/bin/
     )
+
+    SetProjectDependencies(TARGET AppImageKit)
```

Comment:
If I'm not wrong I also saw that somewhere else, so I decided to add it here to keep all recipes consistent.
Can remove these calls across the project of course.